testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionHelper.java

```diff
@@ -195,14 +200,22 @@ public String getContentMessageFile(Subscription sub, int index) {
     }
 
 
-    public Subscription mksub(SubscriptionOrigin origin, DateTime startDate) {
+    public Subscription mksub(SubscriptionOrigin origin, DateTime startDate, SubscriptionPackType packType) {
```

Comment:
I moved SubscriptionHelper & CdrHelper & CsrHelper to a new utils package, and this is still in the kilkari package.
Looks like you haven't merged master.
I can show you how to do it when you make it to the office.